tslib/src/functions/school-pin-generator/random-generator.ts

```diff
@@ -0,0 +1,31 @@
+import crypto from 'crypto'
+export interface IRandomGenerator {
+  generateFromChars (length: number, chars: string): string
+  generateNumberFromRange (minimum: number, maximum: number): number
+}
+export class RandomGenerator implements IRandomGenerator {
+
+  generateNumberFromRange (minimum: number, maximum: number): number {
```

Comment:
It's not clear if the min/max are inclusive or exclusive. E.g.  does `generateNumberFromRange(1,10)` allow 0 and 10 as possible output numbers?